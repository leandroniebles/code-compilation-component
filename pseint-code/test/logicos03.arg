--input=verdadero --input=v --input=1 --input=0 --input=falso --input=f --input=72