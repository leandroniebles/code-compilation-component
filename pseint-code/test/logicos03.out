> verdadero
VERDADERO
> v
VERDADERO
> 1
VERDADERO
> 0
FALSO
> falso
FALSO
> f
FALSO
> 72
=== Line 21: ExeError 120
